code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -3236,3 +3239,48 @@ func (srh *StorageRestHandler) replicateValidatorAggregates(w http.ResponseWrite
 	}
 	common.Respond(w, r, validators, nil)
 }
+
+// swagger:route GET /v1/screst/test/get-active-passed-blobber-rewards-partitions getActivePassedBlobberRewardsPartitions
+// Gets list of validator aggregate records
+//
+// parameters:
+//
+//	+name: period
+//	 description: the period number for which active-passed-blobber-block-rewards partition is requested
+//	 in: query
+//	 type: string
+//
+// responses:
+//
+//	200: StringMap
+//	500:
+func (srh *StorageRestHandler) getActivePassedBlobberRewardsPartitions(w http.ResponseWriter, r *http.Request) {
+	//periodString := r.URL.Query().Get("period")
+	//if periodString == "" {
+	//	err := common.NewErrBadRequest("missing parameter: period")
+	//	common.Respond(w, r, nil, err)
+	//	return
+	//}
+	//
+	//period, err := strconv.ParseInt(periodString, 10, 64)
+	//if err != nil {
+	//	err := common.NewErrBadRequest("invalid period value: " + err.Error())
+	//	common.Respond(w, r, nil, err)
+	//	return
+	//}
+
+	conf, err := getConfig(srh.GetQueryStateContext())
+	if err != nil && err != util.ErrValueNotPresent {
+		common.Respond(w, r, nil, smartcontract.NewErrNoResourceOrErrInternal(err, true, cantGetConfigErrMsg))
+		return
+	}
+
+	ps, err := getCommonActivePassedBlobberRewardsPartitions(srh.GetQueryStateContext(), conf.BlockReward.TriggerPeriod)
```

Comment:
The partitions does not contain the items actually, it only records the total partition number and the partition name.